programs/marginfi/src/instructions/marginfi_group/add_pool.rs

```diff
@@ -325,9 +325,10 @@ pub struct LendingPoolAddBankWithSeed<'info> {
         ],
         bump,
     )]
-    pub fee_vault: Box<Account<'info, TokenAccount>>,
+    pub fee_vault: Box<InterfaceAccount<'info, TokenAccount>>,
 
     pub rent: Sysvar<'info, Rent>,
-    pub token_program: Program<'info, Token>,
+    /// CHECK: Verified manually that its either token or token22
```

Comment:
No manual check has been added in this PR, is this obsolete or a pending todo?

Reply:
Forgot to remove